src/SmartComponents/BaselinesTable/BaselinesTable.js

```diff
@@ -312,36 +234,31 @@ BaselinesTable.propTypes = {
     baselineTableData: PropTypes.array,
     createBaselinesTable: PropTypes.func,
     selectBaseline: PropTypes.func,
-    selectOneBaseline: PropTypes.func,
     addSystemModalOpened: PropTypes.bool,
     fetchBaselines: PropTypes.func,
-    history: PropTypes.object,
+    tableType: PropTypes.string,
+    onClick: PropTypes.func,
     kebab: PropTypes.bool,
     createButton: PropTypes.bool,
     exportButton: PropTypes.bool,
     createBaselineModalOpened: PropTypes.bool,
-    hasSelect: PropTypes.bool,
-    hasSingleSelect: PropTypes.bool,
-    clearSelectedBaselines: PropTypes.func
+    onSelect: PropTypes.func,
+    clearSelectedBaselines: PropTypes.func,
+    columns: PropTypes.array
```

Comment:
Looks like some of the props in this object are now obsolete and could be removed.